app/helpers/view_components/student/editor.rb

```diff
@@ -0,0 +1,16 @@
+module ViewComponents
+  module Student
+    class Editor < ViewComponent
+      initialize_with :solution
+
+      def to_s
+        react_component("student-editor", {
+                          endpoint: Exercism::Routes.api_solution_submissions_path(
+                            solution.uuid,
+                            auth_token: solution.user.auth_tokens.first.to_s
+                          )
+                        })
```

Comment:
Could you drop "student-editor" onto a new line, and then the `{` that follows, then run lint this file pls. Just so it's less long lines and more readable.